src/rpm/package.rs

```diff
@@ -216,31 +208,59 @@ impl RPMPackage {
         );
 
         verifier.verify(header_bytes.as_slice(), signature_header_only)?;
+        self.verify_digests()?;
```

Comment:
Is this intended to always verify digests? If they're missing, the verification would fail and the signatures themselves would already contain the content.

I'd be in favor of providing a on-stop-shop `fn verify` that verifies whatever is in the package.

Reply:
I see, but we do verify both, at least that's what I remember.

Reply:
I don't think so, we verify old signatures and new signatures, but that's on a different dimension from the actual type of signatures.  There's RSAHEADER and DSAHEADER, but we don't have any code that reads DSAHEADER